packages/manager/src/features/VPC/VPCDetail/VPCDetail.tsx

```diff
@@ -1,31 +1,203 @@
+import { EntityHeader } from 'src/components/EntityHeader/EntityHeader';
+import { ErrorState } from 'src/components/ErrorState/ErrorState';
 import { LandingHeader } from 'src/components/LandingHeader';
+import { Paper } from 'src/components/Paper';
+import { useRegionsQuery } from 'src/queries/regions';
+import { useVPCQuery } from 'src/queries/vpcs';
 
-const VPCDetails = () => {
+import { VPCDeleteDialog } from '../VPCLanding/VPCDeleteDialog';
+import { VPCEditDrawer } from '../VPCLanding/VPCEditDrawer';
+
+const VPCDetail = () => {
   const { vpcId } = useParams<{ vpcId: string }>();
+  const theme = useTheme();
+  const { data: vpc, error, isLoading } = useVPCQuery(+vpcId);
+
+  const { data: regions } = useRegionsQuery();
+  const regionLabel = regions?.find((r) => r.id === vpc?.region)?.label ?? '';
+
+  const [selectedVPC, setSelectedVPC] = React.useState<VPC | undefined>();
+
+  const [editVPCDrawerOpen, setEditVPCDrawerOpen] = React.useState(false);
+  const [deleteVPCDialogOpen, setDeleteVPCDialogOpen] = React.useState(false);
+
+  const handleEditVPC = (vpc: VPC) => {
+    setSelectedVPC(vpc);
+    setEditVPCDrawerOpen(true);
+  };
+
+  const handleDeleteVPC = (vpc: VPC) => {
+    setSelectedVPC(vpc);
+    setDeleteVPCDialogOpen(true);
+  };
+
+  if (isLoading) {
+    return <CircleProgress />;
+  }
+
+  if (error) {
+    return (
+      <ErrorState errorText="There was a problem retrieving your VPC. Please try again." />
+    );
+  }
+
+  const numLinodes = vpc?.subnets.reduce(
+    (acc, subnet) => acc + subnet.linodes.length,
+    0
+  );
+
+  const summaryData = [
+    [
+      {
+        label: 'Subnets',
+        value: vpc?.subnets.length,
+      },
+      {
+        label: 'Linodes',
+        value: numLinodes,
+      },
+    ],
+    [
+      {
+        label: 'Region',
+        value: regionLabel,
+      },
+      {
+        label: 'VPC ID',
+        value: vpc?.id,
+      },
+    ],
+    [
+      {
+        label: 'Created',
+        value: vpc?.created,
+      },
+
+      {
+        label: 'Updated',
+        value: vpc?.updated,
+      },
+    ],
+  ];
+
   return (
     <>
       <DocumentTitleSegment segment="VPC" />
       <LandingHeader
         breadcrumbProps={{
           crumbOverrides: [
             {
-              label: 'VPC',
+              label: 'Virtual Private Cloud (VPC)',
               position: 1,
             },
           ],
           labelOptions: { noCap: true },
-          pathname: `/vpc/${vpcId}`, // TODO: VPC - use vpc label, not id
+          pathname: `/vpc/${vpc?.label}`,
         }}
         docsLabel="Docs"
-        docsLink="" // TODO: VPC - Add docs link
+        docsLink="#" // TODO: VPC - Add docs link
+      />
+      <EntityHeader>
+        <Box>
+          <Typography
+            sx={{
+              color: theme.textColors.headlineStatic,
+              fontFamily: theme.font.bold,
+              fontSize: '1rem',
+              padding: '6px 16px',
+            }}
```

Comment:
Let's do this so we don't need the `useTheme` hook.

```suggestion
            sx={(theme) => ({
              color: theme.textColors.headlineStatic,
              fontFamily: theme.font.bold,
              fontSize: '1rem',
              padding: '6px 16px',
            })}
```